Nested import of a file that isn't exported

### Describe the bug

This patch is needed to get tsed/graphql-ws to build using ESModules, because the nested import isn't allowed. 

node_modules/@tsed/graphql-ws/lib/types/GraphQLWSOptions.d.ts:1:31 - error TS2307: Cannot find module 'graphql-ws/lib/server' or its corresponding type declarations.

1 import { ServerOptions } from "graphql-ws/lib/server";
                                ~~~~~~~~~~~~~~~~~~~~~~~


Found 1 error in node_modules/@tsed/graphql-ws/lib/types/GraphQLWSOptions.d.ts:1

### To Reproduce

1. Install TSED 8.7.1 including tsed/graphql-ws 8.7.1 and "graphql-ws": "5.16.0". The package.json file in tsed/graphql-ws 8.7.1 has "graphql-ws": "5.16.0". Then run npm run build.

### Expected behavior

It should build without issue.

### Code snippets

```TypeScript

```

### Repository URL example

_No response_

### OS

macOS

### Node version

v22.14.0

### Library version

v8.7.1

### Additional context

_No response_

--- a/packages/graphql/graphql-ws/src/GraphQLWSModule.ts
+++ b/packages/graphql/graphql-ws/src/GraphQLWSModule.ts
@@ -2,7 +2,6 @@ import Http from "node:http";
 import Https from "node:https";
 
 import {Constant, Inject, InjectorService, Module} from "@tsed/di";
-import type {Disposable} from "graphql-ws";
 import {useServer} from "graphql-ws/lib/use/ws";
 
 import {GraphQLWSOptions} from "./GraphQLWSOptions.js";

--- a/packages/graphql/graphql-ws/src/GraphQLWSOptions.ts
+++ b/packages/graphql/graphql-ws/src/GraphQLWSOptions.ts
@@ -1,6 +1,7 @@
-// @ts-ignore
-import {ServerOptions} from "graphql-ws/lib/server";
-import {ServerOptions as WSServerOptions} from "ws";
+import type {useServer} from "graphql-ws/lib/use/ws";
+import type {ServerOptions as WSServerOptions} from "ws";
+
+type ServerOptions = Parameters<typeof useServer>[0];
 
 export interface GraphQLWSOptions {
   path: string;

